script.js

```diff
@@ -0,0 +1,273 @@
+    EO.preventDefault();
+    let el,
+        key,
+        textarea = document.getElementById("result"),
+        lang = localStorage.getItem('lang'),
+        divs;
+    pressed.add(EO.key);
+    if ( pressed.has("Shift") && pressed.has("Alt") ) {
+      lang = (lang == 'en') ? 'ru' : 'en';
+      localStorage.setItem('lang', lang);
+      let divs = document.getElementsByClassName('key');
+      for (let cell of divs) {
+        let className = cell.firstChild.className;
+        cell.firstChild.className = cell.lastChild.className;
+        cell.lastChild.className = className;
+      }
+    }
+  
+    switch (EO.key) {
+      case 'Alt':
+      case 'Meta':
+      case 'Control':
+      case 'ArrowUp':
+      case 'ArrowRight':
+      case 'ArrowDown':
+      case 'ArrowLeft':
+        break;
+      case 'Delete':
+        
+        break;
+      case 'CapsLock':
+        divs = document.getElementsByClassName('case');
+        for (let cell of divs) {
+          cell.className = (cell.className == 'case up') ? 'case down' : 'case up';
+        }
+        capslock = !capslock;
+        break;
+      case 'Tab':
+        textarea.value += '      ';
+        break;
+      case 'Backspace':
+        textarea.value = textarea.value.substring(0, textarea.value.length - 1);
+        break;
+      case ' ':
+        textarea.value += ' ';
+        break;
+      case 'Enter':
+        textarea.value += '\n';
+        break;
+      case 'Shift':
+        divs = document.getElementsByClassName('case');
+        for (let cell of divs) {
+          cell.className = (cell.className == 'case up') ? 'case down' : 'case up';
+        }
+        break;
+      default:  
+        let strTextArea;
+        for ( el in Keyboard) {  
+          for (let i = 0; i < Keyboard[el].length; i++) {
+            if (EO.keyCode == Keyboard[el][i]['keyCode']) {
+              if (( !pressed.has("Shift") && capslock ) || ( pressed.has("Shift") && !capslock )) {
+                strTextArea = (lang == 'en') ? Keyboard[el][i]['simbols'][3] : Keyboard[el][i]['simbols'][1];
+              } else if ((!pressed.has("Shift") && !capslock) || (pressed.has("Shift") && capslock)) {
+                strTextArea = (lang == 'en') ? Keyboard[el][i]['simbols'][2] : Keyboard[el][i]['simbols'][0];
+              }
+              textarea.value += strTextArea;
+            }
+          }
+        }
+        break;
+    }
+  
+  key = document.getElementsByClassName(EO.code);
+  if ( key[0].parentNode.className.indexOf(' active') == -1) {
+    let str = key[0].parentNode.className + " active";
+    key[0].parentNode.className = str;
+  }
+});
+
+window.addEventListener("keyup", function(EO) {
+	EO = EO || window.event;
+    EO.preventDefault();
+    let el,
+        key,
+        textarea = document.getElementById("result");
+    for ( el in Keyboard) {  
+      for (let i = 0; i < Keyboard[el].length; i++) {
+      }
+    }
+  
+  switch (EO.key) {
+      case 'Shift':
```

Comment:
Would be better if you extract to ```if``` condition. You uset switchcase for one statement